webpack.conf.js

```diff
@@ -1,6 +1,12 @@
 const webpack = require("webpack");
 const _ = require('lodash');
 const ManifestPlugin = require('webpack-manifest-plugin');
+const CleanWebpackPlugin = require('clean-webpack-plugin');
+
+// the path(s) that should be cleaned
+var pathsToClean = [
+  'app/js/.*-bundle-.*'
```

Comment:
This worked once and then stopped working. :(

Reply:
I changed it to a glob (rather than a regex) and it works (though only when starting or restarting gulp): `app/js/*-bundle-*`